migrator/migration-steps.js

```diff
@@ -159,7 +159,7 @@ async function deleteUnusedDirectories (directoriesToDelete) {
       return true
     }
     const reporter = await addReporter(`Remove unused directory ${dir}`)
-    const result = await deleteDirectory(dirPath, { recursive: true })
+    const result = await deleteDirectory(dirPath)
```

Comment:
@BenSurgisonGDS @nataliecarey Think this was a typo as options aren't accepted here

Reply:
Moved to https://github.com/alphagov/govuk-prototype-kit/pull/2385